frontend/src/forms/manage-restaurant-form/ManageRestaurantForm.tsx

```diff
@@ -101,7 +104,10 @@ const ManageRestaurantForm = ({ restaurant, onSave, isLoading }: Props) => {
     });
     formDataJson.menuItems.forEach((menuItem, index) => {
       formData.append(`menuItems[${index}][name]`, menuItem.name);
-      formData.append(`menuItems[${index}][price]`, menuItem.price.toString());
+      formData.append(
+        `menuItems[${index}][price]`,
+        (menuItem.price * 100).toFixed(2).toString()
+      );
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/47#issuecomment-2430019945>#d33549</a><span> - 10/22/2024, 07:15 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Consider using `Number.toFixed()` instead of multiplying by 100 and then using `toFixed(2)`. This might lead to precision issues in some cases.
</div>
<div id="fix">
<b>Fix:</b> Use `Number(menuItem.price).toFixed(2)` instead of `(menuItem.price * 100).toFixed(2)`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -107,4 +107,4 @@
        formData.append(
          `menuItems[${index}][price]`,
 -        (menuItem.price * 100).toFixed(2).toString()
 +        Number(menuItem.price).toFixed(2).toString()
        );
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged